index.bs

```diff
@@ -894,6 +912,65 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
   </pre>
 </div>
 
+## Update camera focus distance and {{takePhoto()}} ## {#example4}
+
+<div class="example" highlight="javascript">
+  <pre>
+    &lt;html>
+    &lt;body>
+    &lt;video autoplay>&lt;/video>
+    &lt;img>
+    &lt;input type="range" hidden>
+    &lt;script>
+      var imageCapture;
+
+      navigator.mediaDevices.getUserMedia({video: true})
+        .then(gotMedia)
+        .catch(err => console.error('getUserMedia() failed: ', err));
+
+      function gotMedia(mediastream) {
+        const video = document.querySelector('video');
+        video.srcObject = mediastream;
+
+        const track = mediastream.getVideoTracks()[0];
+        imageCapture = new ImageCapture(track);
+
+        const capabilities = track.getCapabilities()
+        // Check whether focus distance is supported or not.
+        if (!capabilities.focusDistance) {
+          return;
+        }
+
+        // Map focus distance to a slider element.
+        const input = document.querySelector('input[type="range"]');
+        input.min = capabilities.focusDistance.min;
+        input.max = capabilities.focusDistance.max;
+        input.step = capabilities.focusDistance.step;
+        input.value = track.getSettings().focusDistance;
+
+        input.oninput = function(event) {
+          track.applyConstraints({
+            advanced : [{focusMode: "manual", focusDistance: event.target.value}]
```

Comment:
@yellowdoge Limitations in an implementation should not limit what we put in spec examples IMHO. Specs drive implementations, not the other way around. I.e. our target audience is implementers. For instance, the [WebRTC examples](http://w3c.github.io/webrtc-pc/#advanced-peer-to-peer-example-with-warm-up) contain code no-one has implemented yet. Web developers will hopefully have blogs to rely on for things that work today.

`advanced` [should be avoided](https://github.com/w3c/mediacapture-main/issues/455) where possible as its semantic complexity is higher than needed.